packages/framework/client-logger/fluid-telemetry/package.json

```diff
@@ -94,10 +104,13 @@
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.15.2",
+		"@fluid-experimental/tree": "workspace:~",
```

Comment:
This is the old shared-tree, not the new one. I'm assuming we want to use the new one here?